projects/vdk-heartbeat/src/vdk/internal/heartbeat/job_controller.py

```diff
@@ -68,16 +68,17 @@ def _execute(self, command):
 
     @LogDecorator(log)
     def login(self):
-        self._execute(
-            [
-                "login",
-                "-a",
-                f"{self.config.vdkcli_api_refresh_token}",
-                "-t",
-                "api-token",
-            ]
-            + self.__get_api_token_authorization_url_arg()
-        )
+        if self.config.vdkcli_api_refresh_token:
+            self._execute(
+                [
+                    "login",
+                    "-a",
+                    f"{self.config.vdkcli_api_refresh_token}",
+                    "-t",
+                    "api-token",
+                ]
+                + self.__get_api_token_authorization_url_arg()
+            )
```

Comment:
```suggestion
        if self.config.vdkcli_api_refresh_token:
            self._execute(
                [
                    "login",
                    "-a",
                    f"{self.config.vdkcli_api_refresh_token}",
                    "-t",
                    "api-token",
                ]
                + self.__get_api_token_authorization_url_arg()
            )
        else:
            log.info("Running against local control service.")
```

Maybe include a log statement.

Reply:
+1 for log statement. But not local  - running against control service without authentication